apps/settings/style/icc.css

```diff
@@ -0,0 +1,29 @@
+#icc-entries[data-custom-icon] li > a {
+    padding-left: 5.5rem;
+}
+
+#icc-stk-list[data-custom-icon] li > a {
+    padding-left: 7rem;
+}
+
+#icc-entries[data-custom-icon] li > span,
+#icc-stk-list[data-custom-icon] li > span {
+    display: block;
+    position: absolute;
+    width: 3rem;
+    height: 3rem;
+    left: 1.5rem;
+    top: calc(50% - 1.5rem);
+    box-sizing: content-box;
+    padding: 0;
+}
+
+#icc-stk-list[data-custom-icon] li > span {
+    left: 3rem;
+}
+
+#icc-entries[data-custom-icon] li > span > canvas,
+#icc-stk-list[data-custom-icon] li > span > canvas {
+    margin: auto;
+    display: block;
+}
```

Comment:
Because we are trying to make rtl support better from 2.2+, please help to update these styles to make them fit in rtl languages. I did try it but it looks bad, so we need to tweak some styles here.